src/Importify/Common.hs

```diff
@@ -155,3 +157,23 @@ importSlice []               = Nothing
 importSlice [ImportDecl{..}] = Just $ startAndEndLines importAnn
 importSlice (x:y:xs)         = Just $ startAndEndLines
                                     $ combSpanInfo (importAnn x) (importAnn $ NE.last (y :| xs))
+
+-- | Returns module name of the source file
+-- We can't parse the whole file to get it because default extensions are not available yet
+getModuleName :: Text -> String
+getModuleName src = case impl of
+                        [mn] -> L.init mn
+                        []   -> error "File doesn't have `module' declaration"
```

Comment:
As a side-note: these usages or `error` are okay because modules inside `library` should have only one module signature. But I wonder, is it okay to have such function... Probably there's better way to get module name. And I think it shouldn't be retrieved by this way. Someone can write:

```haskell
module
MyCoolModernHipsterLibWithHipsterModuleSignature
where
```

Well, no sane man will write it in this way but still...